onboard/src/pages/CongratulationsPage.css

```diff
@@ -4,3 +4,8 @@
   left: 5%;
   bottom: 5%;
 }
+.Cake {
```

Comment:
classes should generally speaking be lower case and kebab case. so this should be '.cake'